processing/src/main/java/io/druid/query/filter/DruidDoublePredicate.java

```diff
@@ -19,12 +19,36 @@
 
 package io.druid.query.filter;
 
-
+/**
+ * DoublePredicate is only supported in Java 8+, so use this to avoid boxing when a double predicate is needed.
+ */
+// All implementations are currently lambda expressions and intellij inspections wrongly complains about unused variable.
```

Comment:
Add a link to https://youtrack.jetbrains.com/issue/IDEA-191743 (when this ticket is fixed, annotation and comment could be removed)

Reply:
done. removed stale comment and added note.